### PR TITLE
build: add maven plugins required for an oss release

### PR DESCRIPTION
**Pull request by elek**
README is also updated.

### CODE REVIEW

**Review by mniewrzal — DISMISSED** [review]

**Comment by Erikvv** [comment]
I tested with installing 1.0.0 from remote and it worked.

File: README.md [hunk 1]
```diff
@@ -78,4 +78,36 @@ Example for basic operations.
 
         }
 
-```
\ No newline at end of file
+```
+
+## Development
```
Comment (Erikvv) [inline README.md:83]: Could you explicitly add that this section is for uplink-java maintainers?

File: README.md [hunk 2]
```diff
@@ -78,4 +78,36 @@ Example for basic operations.
...
+    </server>
+  </servers>
+</settings>
+```
+
+Upload to maven staging:
+
+```
+mvn clean deploy -P release
+```
```
Comment (Erikvv) [inline README.md:113]: How does the maintainer mark the release as final?

**Review by Erikvv — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (elek) [reply README.md:113]: As I wrote, it directly releases everything as final. (That's how I accidentally released this version)

File: README.md (hunk 1 above)
Comment (elek) [reply README.md:83]: sure, done.

**Review by Erikvv — APPROVED** [review]

**Review by mniewrzal — APPROVED** [review]

**Pull request merged**